app/controllers/dashboard.js

```diff
@@ -17,6 +18,7 @@ export default Controller.extend({
     this._super(...arguments);
     const defaultType = this._getDefaultType();
     set(this, 'streamType', get(this, 'lastUsed.feedType') || defaultType);
+    this.sticky = new Sticky('.is-sticky');
```

Comment:
Since we don't have a post-insert hook available in Controllers, this should be wrapped in an `afterRender` schedule.